pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/creator/impl/inv/RangeIndexCreator.java

```diff
@@ -112,12 +112,31 @@
   public RangeIndexCreator(File indexDir, FieldSpec fieldSpec, DataType valueType, int numRanges, int numValuesPerRange,
       int numDocs, int numValues)
       throws IOException {
```

Comment:
Note that this is the old range index creator, and won’t be used by default from 0.10.0 onwards

Reply:
true, I actually don't need to modify this piece of code. Will revert.